tests/board-builder/new-board.test.ts

```diff
@@ -0,0 +1,15 @@
+import test from "ava"
+import { getTestFixture } from "tests/fixtures/get-test-fixture"
+
+test("[smoke] new board test", async (t) => {
+  const { pb, logSoup } = getTestFixture(t)
+
+  const soup = await pb
+    .add("board", (bb) => {
+      bb.setProps({ width: 10, height: 10, pcbX: 20, pcbY: 0 })
+    })
+    .build()
+
```

Comment:
Can you use su(soup) to add a condition and check the board to make sure that the boards position is correct.

When its possible to create conditions thats preferable to prevent breakage